backend/internal/consumer/storage_updater.go

```diff
@@ -21,7 +21,8 @@ type storageUpdater struct {
 	wg *sync.WaitGroup
 }
 
-func newDatabaseUpdater(storer storage.Storer, msgChannel msgChannelType, closeChannel signalChannelType, wg *sync.WaitGroup) *storageUpdater {
+func newDatabaseUpdater(storer storage.Storer, msgChannel msgChannelType, closeChannel signalChannelType,
+	wg *sync.WaitGroup) *storageUpdater {
```

Comment:
Seems like an unnecessary change, please revert

Reply:
My gofmt lint auto did that. I try to put it back to norm but after save it auto lint again.

Reply:
Try changing this to 
```golang
func newDatabaseUpdater(
	storer storage.Storer,
	msgChannel msgChannelType,
	closeChannel signalChannelType,
	wg *sync.WaitGroup,
) *storageUpdater {
```
so it looks clearer